src/frontend/components/Home/Home.js

```diff
@@ -18,8 +18,9 @@ export default class Home extends React.Component {
         <main>
           <div className="search-background">
             <div className="search-title">
-              <h2><em>"Ready for</em> </h2>
-              <h2><em>Pair Programming"</em></h2>
+              <h1>Start</h1>
+              <p>Find a user</p>
+              <p>Request a pair Programming</p>
```

Comment:
Can we change the slogan to "Let's Pair Program ! Find your coding partner "

Reply:
Won't a bit former sentences suit to the theme??